app/views/services/show.html.haml

```diff
@@ -18,4 +18,4 @@
         = t('.actions')
 
   %tbody
-    = render partial: 'environment', collection: @status_by_environment
+    = render partial: 'environment', collection:  @status_by_environment
```

Comment:
looks like a random space has crept in - if you take it out, then this file will remove itself from this diff & PR (which is a good thing)